packages/core/src/logger/loggerConfiguration.ts

```diff
@@ -1,15 +1,19 @@
-export interface LoggingConfiguration {
-  isLoggingRequestInfo: boolean;
-  isLoggingRequestHeaders: boolean;
-  isLoggingRequestBody: boolean;
-  isLoggingResponseInfo: boolean;
-  isLoggingResponseHeaders: boolean;
-  isLoggingResponseBody: boolean;
-  headerFilters: string[];
-  headerLoggingPolicy: HeaderLoggingPolicy;
+import { Level, LoggerInterface } from '@apimatic/core-interfaces';
+
+export interface LoggingOptions {
+  logger?: LoggerInterface;
+  logLevel?: Level;
+  logRequest?: LogRequestOptions;
+  logResponse?: LogBaseOptions;
+}
+
+export interface LogBaseOptions {
+  logBody?: boolean;
+  logHeaders?: boolean;
+  headerToExclude?: string[];
+  headerToInclude?: string[];
 }
 
-export enum HeaderLoggingPolicy {
-  Include = 'include',
-  Exculde = 'exclude',
+export interface LogRequestOptions extends LogBaseOptions {
```

Comment:
```suggestion
export interface HttpRequestLoggingOptions extends HttpMessageLoggingOptions {
```

Reply:
All of the interfaces names were updated